### PR TITLE
[Portuguese] Fix for translation of "second".

### PR DESCRIPTION
**Pull request by asadurski**
Quick fix for issue https://github.com/scrapinghub/dateparser/issues/113.

### CODE REVIEW

File: tests/test_freshness_date_parser.py [hunk 1]
```diff
@@ -107,6 +107,7 @@ def setUp(self):
         param('anteontem', ago={'days': 2}, period='day'),
         param('hoje', ago={'days': 0}, period='day'),
         param('uma hora atrás', ago={'hours': 1}, period='day'),
+        param('1 segundo atrás', ago={'seconds': 1}, period='day'),
```
Comment (VOIDCRUSHER) [inline tests/test_freshness_date_parser.py]: Should there also be another check in test_translation under the Portuguese section?

```python
param('pt', "1 segundo atrás", "1 second ago"),
```

**Comment by VOIDCRUSHER** [comment]
LGTM

File: tests/test_freshness_date_parser.py (hunk 1 above)
Comment (asadurski) [inline tests/test_freshness_date_parser.py]: Oh yes, that wouldn't hurt. Thank you.

**Comment by waqasshabbir** [comment]
LGTM! let's add `Cesar Flores` VOIDCRUSHER to `AUTHORS` and merge it.

**Pull request merged**